core/src/main/java/com/netflix/conductor/service/MetadataService.java

```diff
@@ -155,9 +155,26 @@ public void registerWorkflowDef(WorkflowDef def) {
 		metadata.create(def);
 	}
 
+    /**
+     *
+     * @param name Name of the workflow definition to be removed
+	 * @param version Version of the workflow definition.
+     */
+	public void unregisterWorkflowDef(String name, int version) {
+        if (name == null) {
+            throw new ApplicationException(Code.INVALID_INPUT, "Workflow name cannot be null");
+        }
+
+        if (version == 0) {
+            throw new ApplicationException(Code.INVALID_INPUT, "Version is not valid");
+        }
+
+	    metadata.removeWorkflowDef(name, version);
```

Comment:
fix indent

Reply:
Looks like my indentation setting is different will fix it.